stories/components/Todo/TodoListItem.js

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types'
+import React, { PureComponent } from 'react'
+import Button from '@helpscout/hsds-react/components/Button'
+import Card from '@helpscout/hsds-react/components/Card'
+import Flexy from '@helpscout/hsds-react/components/Flexy'
+import connect from '../../../src/components/connect'
+
+class TodoListItem extends PureComponent {
+  static propTypes = {
+    removeTodo: PropTypes.func,
+  }
+
+  static defaultProps = {
+    removeTodo: () => {}
+  }
+
+  static displayName = 'TodoListItem'
+
+  render() {
+    const { removeTodo, todo: { id, task }, ...rest } = this.props
+
+    return (
+      <Card.Block {...rest}>
+        <Flexy>
+          <Flexy.Block>{task}</Flexy.Block>
+          <Flexy.Item>
+            <Button onClick={() => removeTodo(id)}>Remove</Button>
+          </Flexy.Item>
+        </Flexy>
+      </Card.Block>
+    )
+  }
+}
+
+const actions = store => {
+  const { removeTodo } = store.getExternalActions()
+  return {
+    removeTodo: (_state, id) => removeTodo(id)
```

Comment:
Is this a convention for naming ignored lambda/anonymous function parameter?

Reply:
All the code in the `stories` directory serves as an example. I just wanted to highlight that we were remapping this function to toss away the first argument. I'll just put a comment here instead so as not to be confusing with the `_` prefix.

Reply:
I mean - I understood what it was doing, I just wasn't 100% sure :)